### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by monperrus**

### CODE REVIEW

**Comment by danglotb** [comment]
@monperrus I just updated the README, WDYT?

File: README.md [hunk 1]
```diff
@@ -20,20 +20,138 @@ This project provides a framework to run JUnit test in a new JVM. It allows to r
...
+The only API that you should use is `eu.stamp_project.testrunner.EntryPoint`.
+
+This class provides several methods to execute all the test methods of given test classes, specific test methods of a given test class, compute the code coverage with JaCoCo, etc.
+
+#### Tests Execution
+
+1. Executing all the test methods of a test class.
+
+```java
+EntryPoint.runTests(String classpath, String fullQualifiedNameOfTestClass);
```
Comment (monperrus) [inline README.md]: How to retrieve the test results?

**Comment by monperrus** [comment]
Cool, thanks.

What about renaming TestListener to TestResult?

Could you add links to the source code of EntryPoint and TestListener?

File: README.md [hunk 2]
```diff
@@ -20,20 +20,131 @@ This project provides a framework to run JUnit test in a new JVM. It allows to r
...
+##### Output
+
+The output of all `runTests()` API is a `eu.stamp_project.testrunner.listener.TestListener`.
+
+This object provides all the information needed about the execution of test methods:
+
+   * `getRunningTests()`: the list of test methods that have been executed.
+   * `getPassingTests()`: the list of test methods that succeed.
+   * `getFailingTests()`: the list of test methods that failed.
+   * `getAssumptionFailingTests()`: the list of test methods that have a failing assumption.
```
Comment (monperrus) [inline README.md]: what is a failing assumption? (explain and/or add a link?)

File: README.md [hunk 3]
```diff
@@ -20,20 +20,131 @@ This project provides a framework to run JUnit test in a new JVM. It allows to r
...
+   * `getIgnoredTests()`: the list of test methods that are ignored.
+   
+The method `TestListener#aggregate(TestListener that)` allows to aggregate the results.
```
Comment (monperrus) [inline README.md]: what does it return?

**Comment by danglotb** [comment]
WDYT? Is it clear enough?

**Comment by danglotb** [comment]
@monperrus okay to merge?

**Comment by monperrus** [comment]
yes

**Pull request merged**